### PR TITLE
Medicine balancing

### PR DESCRIPTION
**Pull request by cadyn**
Can thank @LunarFleet for the ideas. I have a quote from regarding the reasoning of these changes.

Changes:
Osteodaxon no longer mends fractures, instead it temporarily negates the effects of all fractures, allowing the user to use such organs as if they were not broken.
Osteodaxon also metabolizes much more slowly now, and its recipe has been changed to the following: 10 Inaprovaline + 20 Bicaridine + 20 Paracetamol + 10 Calcium + 3 Phoron + 5 Phoron (Catalyst) = 60u Osteodaxon
Myelamine no longer heals internal bleeding, instead temporarily slowing bleeding bleeding by a factor of 90% on average, dependent on the species chem effectiveness variable.
Myelamine also metabolizes much more slowly, and its recipe has been changed to the following: 20 Bicaridine + 20 Inaprovaline + 20 Iron + 3 Phoron + 5 Phoron (Catalyst) = 60u Myelamine
Peridaxon no longer heals organs, instead it reverses any necrotic organs, both internal external

Lunar's quote:

> Osteodaxon now serves as a field solution for broken bones, instead of being a cure-all for any sort of bone damage. When on osteodaxon, you are effectively able to ignore broken bones until the chemical wears off. This makes it a very good chem for expeditions, whilst not invalidating the rest of medical.

> Myelamine slows bleeding to a near completely negligible amount, similarly allowing expeditions to go on uninterrupted, same as osteodaxon's new purpose. By slowing bleed, this also makes it an effective stabilizing chemical. The end intent is the  same; make it so osteo and myelamine cure the symptoms of a problem instead of curing the problems themselves, as these two chems, in their currently live state, invalidate two major parts of medical.

> Now, for Peridaxon, well Contray to popular belief, there are fourish other kinds of -axons which are designed SPECIFICALLY to heal sets of organs.However, none of these are made, hell, even heard of, because people just make peridaxon as a cure all instead. NOW, peridaxon can be used to fix a previously incurable problem; necrotic organs, which, normally, would only be able to b e fixed via organ transplantation

### CODE REVIEW

**Comment by Muffindrake** [comment]
I'd also cut down on the inevitable Bicaridine overdose internal bleeding heal powergaming, at least early on - remove the Carbon pill bottles from the nanomed vendors. They contain 840u of Carbon per vendor, which I don't believe anyone but powergamers-in-denial use to refill the Carbon cartridges in their dispensers. This would be a band-aid to stop people from trying to game the medical system by avoiding Medical entirely, unless they need the odd break fixed, early-game.

I've been told by some people on a server with a very similar code base that exploration really needs their overpowered chems to stay as they are, but I don't believe that to be the case. All in all, I don't respect game design advice from people that ask me to make 1:100 osteodaxon/water mixes and pretend it's not powergaming.

As it is right now, Peridaxon does way too much for a single chemical. If it's not entirely changed to reverse necrosis like this PR on repaired tissue, it should be at least changed to be unable to mend ruptured lungs, which should be reserved for Respirodaxon, and be unable to heal organs above a certain damage threshold, like 10.

Having ways to slow, and with myelamine/inaprovaline/OD bicaridine, slowly heal internal bleeding is fine. At least IB won't be an entire death sentence if it occurs, whereas now, you stab someone with a myelamine injector and it's k.

In addition to have myelamine in this form available, it's also neat to have osteodaxon ignore the effects of broken bones. That means that someone with a break isn't immediately crippled by having to wear splints until they get to medical, yes?

**Comment by Wickedtemp** [comment]
TLDR: None of this is actually necessary, sounds like Lunar's just irritated at Explo if the fact that its the primary example for the majority of the changes is anything to go on. I'll go point by point. 

Credentials time! I am Wickedtemp. I've been playing ss13 since late 2014, and have played regularly since then. Medical is all I do. I hyperfocused on Medical so much that I OOCly don't even know how to make a stun-prod or crossbow or hack doors, but I was able to write this: https://docs.google.com/document/d/1_zqIZgqhRDxC0EVg-ndUeY-YGZ-D11kvxggAGUC_GaE/edit?usp=sharing Now, I understand that for a while, Chompers has been going through a transition to Baycode, so there may be differences in how the Medical system functions. You guys know more about your dev situation than I do, so do whatever you want with this information because it's from a perspective of PolarisCode Medical. 

"""Osteodaxon no longer mends fractures, instead it temporarily negates the effects of all fractures, allowing the user to use such organs as if they were not broken."""
- More or less the same, but they'll still need surgery later, then? I'd rather keep it as it is. It isn't made often due to its recipe requiring carp toxin, If it's used within Medbay, Osteodaxon is one of those things that we're just kinda glad we managed to get, especially if the patient had 4+ fractures. It's one of those FeelGood moments, and I don't see why it should be modified and lessened over Exploration getting their hands on it. There are a number of other ways to rebalance while still keeping a potent and useful effect. The phoron cost in the new recipe also seems high for what's essentially a glorified painkiller/splint combo.


"""Myelamine no longer heals internal bleeding, instead temporarily slowing bleeding by a factor of 90% on average, dependent on the species chem effectiveness variable.""" 
- The other primary change with exploration as the driving example for its necessity. The thing is, we can already do this rather easily with bicaridine overdoses, and the only thing this change will result in is an increase in bicaridine overdoses. You might be thinking "Oh, hey, that's still more resources they'll have to carry and use!" but uh... not really. All they gotta do is take a pill bottle and a bit of extra dylovene and it's ezpz. You might be thinking, "But Tempest! What about Chemistry? They can refuse to make a bunch of extra bicaridine for OD's!" -- you're right. But... I wouldn't. It's not like it's hard to pump out 20 bottles of bicaridine. I usually make around 12 a round as it is, so what do I care? So, they get all the resources they need with zero extra hassle. Myelamine is just a lil more convenient, patient-friendly, and comes with a purple kit which is a bonus because it's my favorite color. 

"""Peridaxon no longer heals organs, instead it reverses any necrotic organs, both internal external"""
- This is the biggest "dude, what even" change. Peridaxon has been roughly unchanged for 5+ years. The only changes that were added was a blurry-vision/dizziness side effect so you'd screw yourself over if you tried to use it in an active combat situation. Peridaxon's niche is more focused on repairing multiple organs at once. It's extremely slow, and if the patient has more than 5 damage on an organ, it's literally faster to just operate if you're in Medbay. So... you're removing the entire point of making peridaxon, in favor of... ... ...necrosis organ healing. Now, credit where credit is due, there's no real way to heal an internal organ afflicted with necrosis (if we're talking about the infection, anyway). Generally, you'd have to just do a transplant, but we have a bioprinter for that exact reason, and we can just add this effect to, say, a special reaction when corophizine metabolizes in the presence of peridaxon and call it a day. For external organs, there's... a surgery for that already, and it involves peridaxon. So... why? And for something that rarely, RARELY happens because nine times out of ten, the patient has their infection cleared up, or they die, long before it reaches necrosis stages. 

And... yes, there are other "daxons", but uh... Nobody uses them. They're dumb. Most of them require biomass and generally speaking nobody thinks its worth it to actually make these chems. I told Mechoid that would happen and they didn't really change anything, so now here we are, with those chems being more or less abandoned. Not forgotten, just left behind because why even bother https://github.com/PolarisSS13/Polaris/pull/5846 -- Even if you remove peridaxon, this won't really lead to an increase in production of these other chems. Peri tends to only be used in cases of widespread organ trauma, and lung ruptures. I can stabilize a ruptured lung super easily without peri or respirodaxon, so I'll just drop Peri and stop making it, keep the ruptured lung patient stable, and effectively nothing changes. 

So, that's all for the first post. Onto the next post.

"""I'd also cut down on the inevitable Bicaridine overdose internal bleeding heal powergaming, at least early on - remove the Carbon pill bottles from the nanomed vendors. They contain 840u of Carbon per vendor, which I don't believe anyone but powergamers-in-denial use to refill the Carbon cartridges in their dispensers. This would be a band-aid to stop people from trying to game the medical system by avoiding Medical entirely, unless they need the odd break fixed, early-game."""
- That also won't change anything. They're useful, but I make 12 bottles of bicaridine per round with or without carbon pills, so... my mass-production of reagents won't be reduced. I can also just order a thousand units of carbon from cargo, so... it's whatever. Things will take five minutes longer to make, worst-case. 

"""I've been told by some people on a server with a very similar code base that exploration really needs their overpowered chems to stay as they are, but I don't believe that to be the case. All in all, I don't respect game design advice from people that ask me to make 1:100 osteodaxon/water mixes and pretend it's not powergaming."""
- Full disclosure, I don't bother with the absurdly-diluted osteodaxon mixes. That should probably get fixed. I don't use it for every fracture I come across, it's reserved for cases of three or more fractures, generally speaking. Outside of this, there's no real "overpowered chem D:<" . Peridaxon is slow AF and has side effects that will screw you in active combat. I don't even rely on Myelamine because sometimes it flat out just won't be enough if the internal bleeding is too severe, and I think people who rely on Myelamine instead of injecting it and rushing to Medbay are dumb for doing so. Peridaxon is fine, apparently the only issues you have with it are limited to "exploration uses it to heal ruptured lungs!". Okay? And? It's literally always been able to do that. You know what it doesn't do? It doesn't set lose.breath to 0. So if your ruptured lungs aren't fixed quickly and you use peridaxon, you get fucked over because even though your lungs are healed, you're stuck with suffocation for a good while. 


"""As it is right now, Peridaxon does way too much for a single chemical. If it's not entirely changed to reverse necrosis like this PR on repaired tissue, it should be at least changed to be unable to mend ruptured lungs, which should be reserved for Respirodaxon, and be unable to heal organs above a certain damage threshold, like 10.

Having ways to slow, and with myelamine/inaprovaline/OD bicaridine, slowly heal internal bleeding is fine. At least IB won't be an entire death sentence if it occurs, whereas now, you stab someone with a myelamine injector and it's k.

In addition to have myelamine in this form available, it's also neat to have osteodaxon ignore the effects of broken bones. That means that someone with a break isn't immediately crippled by having to wear splints until they get to medical, yes?"""
- Kinda covered all of this in the previous point. Peri isn't OP and isn't used and relied upon nearly as often as you seem to think it is, it's relatively self-balancing because healing anything over 5 becomes slower than just operating, myelamine isn't nearly as reliable as you think it is and I've been disappointed by it so many times that I don't even expect it to heal the internal bleeding, and...wearing splints isn't really much of a 'crippling' thing.. 


Does anybody have any questions, comments, or concerns?

**Comment by Muffindrake** [comment]
You can deal with people trying to abuse Bicaridine IB heal excessively by reducing the amount of IB damage it heals. Cutting it by 80% would deal with the problem quite effectively, no?

**Comment by Wickedtemp** [comment]
It's an intended game mechanic, so "using it" doesn't qualify as "abuse". 

"People use it" is not abuse. 

It's not good practice to go "oh, this is being relied on too much, I don't like that. Remove it. Oh, people switched to relying on something else instead? Cut it by 80%", generally speaking.

**Comment by Muffindrake** [comment]
> It's an intended game mechanic, so "using it" doesn't qualify as "abuse".

Just because something has been in there for a couple years, doesn't make it not broken. A bug, or a broken item are still that.

>It's not good practice to go blurb

Who said that and why should I listen to them? From what I can see, you are hardly making arguments from cold hard matter, and preferring to make arguments from emotion, or feelings. "I'd rather this be the case" is probably not the way to drive development. Let me list the issues with the current chemicals for you:

- Osteodaxon, Myelamine and Peridaxon invalidate Medical almost entirely. Surgery does literally not exist with them around, barring cases where people get superdead, necrotic organs, or missing organs or limbs
- Peridaxon has no real drawback, can be made in large quantities, metabolizes too quickly for the side-effect to be noticeable in the field, and literally heals every single type of organ damage there exists in this game, including the serious effect of ruptured lungs. There is an entire class of specialized organ healing chemicals that do not get made whatsoever because this abomination exists
- A lot of players are treating Medical as an inconvenience rather than a vital part of the game, and the existence of that form of chemicals does not help this matter

Your dismissive and downright rude attitude towards someone that is trying to make meaningful change (hell, it's a person I don't even know) to stop certain professions from literally circumventing Medical, is strange. You don't start a serious comment with a `TLDR, this lunar person is mad`. You don't dismiss people out of hand with `I'd rather this be the case`. This is not the type of conduct that should be displayed in a project built on code. And on top of that, you start off your post with some pompous credential flexing on a server you don't even play on, as if that had any bearing whatsoever. You're being a clown.

**Comment by LunarFleet** [comment]
I haven't actually read this all but; tempest, why are you even here? I don't think I've seen you play here once. 

I've been a medical main for a good year now, but, I'm not gonna wave that around, if the idea is steadfast, doesn't matter where it came from.

'More or less the same, but they'll still need surgery later, then? I'd rather keep it as it is. It isn't made often due to its recipe requiring carp toxin, If it's used within Medbay, Osteodaxon is one of those things that we're just kinda glad we managed to get, especially if the patient had 4+ fractures. It's one of those FeelGood moments, and I don't see why it should be modified and lessened over Exploration getting their hands on it. There are a number of other ways to rebalance while still keeping a potent and useful effect. The phoron cost in the new recipe also seems high for what's essentially a glorified painkiller/splint combo.'

First off, I have nothing against exploration, in fact, exploration is one of my favorite departments, and I've had a good bit of fun on expeditions. None of this was suggested because 'explo man bad' I am about as far from that kind of person as you can get. This was suggested because these three chems invalidate the rest of medical, science, and these chems are rather flagrantly overpowered. 

'Osteodaxon is one of those things that we're just kinda glad we managed to get, especially if the patient had 4+ fractures.' This is genuinely just power gaming, especially with the mixes I see for osteo. If I remember correctly, 0.1 units heals one fracture. So, one unit heals ten. Like Muffin said; I'm not going to take people seriously when they try to say that making 1-10 or 1-100 osteo mixes and try to say it isn't powergaming, _**It is, whether it is an intended feature or not, an intended feature can still be overpowered and abusive.**_

Lastly, unless Cadyn did not code it right, it ISN'T just splinting; Splinting still slows you and what not. Osteo is intended to completely negate that as well, no slowdown, no bone pain. 

'The other primary change with exploration as the driving example for its necessity. The thing is, we can already do this rather easily with bicaridine overdoses, and the only thing this change will result in is an increase in bicaridine overdoses. You might be thinking "Oh, hey, that's still more resources they'll have to carry and use!" but uh... not really. All they gotta do is take a pill bottle and a bit of extra dylovene and it's ezpz. You might be thinking, "But Tempest! What about Chemistry? They can refuse to make a bunch of extra bicaridine for OD's!" -- you're right. But... I wouldn't. It's not like it's hard to pump out 20 bottles of bicaridine. I usually make around 12 a round as it is, so what do I care? So, they get all the resources they need with zero extra hassle. Myelamine is just a lil more convenient, patient-friendly, and comes with a purple kit which is a bonus because it's my favorite color.'

Bicaridine overdose is something I looked over, but, even then, it is less effective than myelamine, and it deals overdose damage. With Erik's overdose changes, which'll probably land here _eventually_, that is a pretty decent balancing feature. Regardless, could just have bicaridine follow Myel and slow bleeding tremendous.


'This is the biggest "dude, what even" change. Peridaxon has been roughly unchanged for 5+ years. The only changes that were added was a blurry-vision/dizziness side effect so you'd screw yourself over if you tried to use it in an active combat situation. Peridaxon's niche is more focused on repairing multiple organs at once. It's extremely slow, and if the patient has more than 5 damage on an organ, it's literally faster to just operate if you're in Medbay. So... you're removing the entire point of making peridaxon, in favor of... ... ...necrosis organ healing. Now, credit where credit is due, there's no real way to heal an internal organ afflicted with necrosis (if we're talking about the infection, anyway). Generally, you'd have to just do a transplant, but we have a bioprinter for that exact reason, and we can just add this effect to, say, a special reaction when corophizine metabolizes in the presence of peridaxon and call it a day. For external organs, there's... a surgery for that already, and it involves peridaxon. So... why? And for something that rarely, RARELY happens because nine times out of ten, the patient has their infection cleared up, or they die, long before it reaches necrosis stages.'

Because people don't even look at surgery, at all, here at least. I make no comments for your home land of Virgo proper. But HERE, people will dump forty units of peridaxon into someone if it is needed. This, given peri's rate of metabolism, makes it a rather unfun time for the person who now has to sit there for a few minutes. The necrosis organ healing isn't an equivalent exchange, it isn't supposed to be. Peridaxon is the magic heal-ey chem that invalidates the rest of the organ healing chems. The change gives it a specific purpose, like other chems, and leaves it at that. 


'Full disclosure, I don't bother with the absurdly-diluted osteodaxon mixes. '
This is why I ask _why you're even here._ You don't, but, we're not talking about you, we're talking about the rest of the medical mains on CHOMP, a good half of which crutch on diluted osteo mixes any time they can get their hands on them. As I said, many would rather use 40 units of peridaxon to heal one damaged organ, rather than fix it with a minute or two of surgery. 

' Peri isn't OP and isn't used and relied upon nearly as often as you seem to think it is'
Again, I have never seen you play medical here. If I am wrong, I'd be happy to hear what characters you play here for reference. 

**What I'm getting from this all is that you like the chems which make medical easy. Which, is fine, I can understand liking things to be easy. But the 'easy' is invalidating science's contributions and making Chomp become infested with inexperienced medics. And, to be honest, I don't really think your sort of mindset should be writing a medical guide either. You may have years of medical experience, but if those experiences are just 'inject osteo into patient' or 'inject dex+ into patient, wait for myel to work' then that isn't really meaningful.**

Tl;Dr Go home GI

**Comment by Wickedtemp** [comment]
"BUT TEMPEST WHY ARE YOU EVEN HERE?!"
dude you literally asked me to "chime in"
https://gyazo.com/c5b90bb51616b773d9da5f38cb718b0b

Don't ask me to chime in about a topic and get mad when I show up to the table.

**Comment by Razgriz1032** [comment]
Wickedtemp you are wrong... Lunarfleet asked you why you were here, not muffindrake...

**Comment by LunarFleet** [comment]
Indeed, I am not Muffin, I'm Lunar, I play Areli and Clea-Nor. I'm not sure why Muffin dragged you here, as this is a good bit out of your jurisdiction. Whatever is going on in Virgo, I take no part in, nor does this have anything to do with it. This was not suggested because 'exploration man bad', it was suggested because 'Chomp medical mains keep crutching on peridaxon and what not'

**Comment by Muffindrake** [comment]
I assumed they had anything useful to contribute to the conversation, given that their information on Medical contains some amount of specifically chompers-related information. They'd have seen that anyway in Virgo's #medical channel, as I posted that there for discussion, only to get immediately piled on by five powergame-y exploration players.

I had wanted that discussed there, because Medical is still the same between both servers currently, and I incidentally play on both. In both cases, here and their discord, I expected some measured discussion from members of that server, but that doesn't appear to be possible. Even right this instant someone was defending Peridaxon for 'lore reasons'. Ugh.

**Comment by lowpowermia** [comment]
personally i'm just thinkin about how someone could just load themselves up with copious amts of osteodaxon and just using it 100% of the time instead of even thinkin about fixing the bones ever, especially with it bein' so cheap. Myelamine seems pretty whatever, though.

**Comment by LunarFleet** [comment]
Its not really that the systems aren't the same, they are. The issue is that the medical players are different. I used to play on virgo, I don't know what medical is like now, but overall, medical was more intelligent over there. Here, the majority of doctors rely heavily on chemistry, which is why I suggested this PR, to encourage them to learn non-chemistry. Because that chemistry isn't always there, and having a doctor who takes twenty minutes to fix you because they don't know what they're doing without their chems is kinda shitty.

But anyway, to what Mia said. It costs phoron, and if you're going back to chemistry to refill on osteo over and over, you might as well just get the bones fixed to begin with, rather than risking the osteo wearing out while in combat.

**Comment by Wickedtemp** [comment]
@Muffindrake 
""Osteodaxon, Myelamine and Peridaxon invalidate Medical almost entirely. Surgery does literally not exist with them around, barring cases where people get superdead, necrotic organs, or missing organs or limbs
Peridaxon has no real drawback, can be made in large quantities, metabolizes too quickly for the side-effect to be noticeable in the field, and literally heals every single type of organ damage there exists in this game, including the serious effect of ruptured lungs. There is an entire class of specialized organ healing chemicals that do not get made whatsoever because this abomination exists
A lot of players are treating Medical as an inconvenience rather than a vital part of the game, and the existence of that form of chemicals does not help this matter"""

Surgery does exist. The players just aren't choosing to use it. This problem isn't a code problem, it's a playerbase problem. You can try to fix that problem with code changes, it might work, it might not. When the overdose changes hit, it'll be harder to stabilize a patient that you gave 40u peridaxon to, so that might serve as an incentive. The other "daxons" aren't used because there's no real point. Even if you removed peridaxon, the only one I'd even consider making is respirodaxon for lungs, and all of the others remain just as un-used. There are other issues with them other than peridaxon.

As for what Lunar's posted, their post is too lengthy to address point by point, but I'll give a quick series of 1-2 liners.

- I used to play here, for a short time. I don't any longer, but that doesn't mean I'll never pop back in, even if it's been quite a while. I'll probably resume playing here occasionally. I enjoyed the Secret Santa stuff, especially.
- Simply using osteodaxon is not powergaming. Making the diluted mixes, however, is, and I already said that this should be fixed.
- The overdose changes don't hit bicaridine hard enough. Bicaridine overdose is just as viable, and it will be used much more often, because it works and is easy to do.
- The fact that medical players here ignore non-peridaxon methods of healing internal organ trauma is just a side effect of inexperience. If they're interested in learning, you can teach them. You can do this either OOCly from a "surgery would be much faster" standpoint or an IC "overdosing your patient is malpractice, we should do this another way" plea for roleplay/ethics reasons. Same thing for people using cryo to heal everything, or only relying on tricordrazine, etc. The folks who are interested in the department will excitedly learn new things. 

""What I'm getting from this all is that you like the chems which make medical easy. Which, is fine, I can understand liking things to be easy. But the 'easy' is invalidating science's contributions and making Chomp become infested with inexperienced medics. And, to be honest, I don't really think your sort of mindset should be writing a medical guide either. You may have years of medical experience, but if those experiences are just 'inject osteo into patient' or 'inject dex+ into patient, wait for myel to work' then that isn't really meaningful.""

This, I'll respond to, directly. I enjoy chemistry, I enjoy Medical as a whole. I enjoy operating, virology, and I enjoy writing and updating the MedicalGuide that I've been maintaining. I've been playing long before osteodaxon and myelamine were added to this codebase, so I can promise you that my experiences go far, far, far, far, FAR beyond "inject osteo and dex+ into patient and wait for myel to work". I love the gameplay and mechanics, and the roleplay of Medical fits my character the best, out of any other department, so I've stuck with it. I don't use peridaxon unless the organ damage is below 5, on a non-eye/brain/liver organ. If the liver is at 10 or lower, I use dylovene, and if it's 10-25, carthatoline. If the eyes are at 50 or lower, Imidazoline. If the brain is at 30 or lower, I'd recommend alkysine. Any other organ at 6+? I'd recommend operating. For osteodaxon, I only use it if the patient has a number of fractures, and I simply don't rely on myelamine at all. Internal bleeding equates to prioritized operating. Now, all of this can be over-ridden if the patient is not on death's-door, is conscious, and requests a different treatment. Maybe they don't want to be operated on, that's okay, the patient has a say in what medical treatments they're given. Maybe they'd rather surgery than a chem, that's also fine. I make the recommendations, so long as they're conscious and stable, and then the patient decides what to do from there. Most of the time they just go with whatever I recommend because they just want to get fixed. Roleplay definitely factors into it all.  I don't know what Science can produce for Medical, but if it hasn't been drastically changed from Virgo's, they still have plenty to make. Surgical tools, bluespace beakers, mechs, ML3M cells, advanced roller beds, power cells, chem sprayers, and upgraded equipment within Medbay. The surgical tools aren't invalidated because a few reagents exist. If the root problem is "players here aren't making use of surgery when they probably should", then I'd think it'd make the most sense to simply try to teach them why surgery is a good idea sometimes. That's why I wrote the MedicalGuide in the first place, and most of it should still apply to this server, depending on how much has changed in the last two or so years. I apologize that my mentioning of it came off as 'flexing', that wasn't my intention. I believed it was relevant, given that you probably wouldn't know who I was, wouldn't know I used to play here for a short while, and wouldn't know if I actually knew what I was talking about when it came to Medical as a whole. 

If you have any questions, comments, or concerns, my discord is Wickedtemp#5642. Feel free to message and stuff.

**Comment by LunarFleet** [comment]
Surgery does exist, doesn't mean it is used. The point of this isn't to limit the playerbase into using a singular solution. They don't have to make the other organ meds, they can use surgery. And if they don't want to use surgery, they  can use the other chems. But if one chem is overpowered enough to make all the others meaningless, there is a problem. That is the state of peridaxon, its lazy and simple. Though, I could have worded the 40 unit thing better. I don't mean they inject 40 units all at once. No no, they'll stay under the OD limit, but they'll keep someone in the lobby until 40 fucking units metabolize. People treat medical as a chore, not a role they want to play, that is the root issue. 

First, I am aware that you have played here before, but it wasn't any time recently. Due to outside sources, Chomp has gotten an influx of new players, which you have not met or interacted with, for the most part. As you and I have said, this is a matter of the playerbase. You may know the medical system, you do not yet know the playerbase.

Second, you're right, simply using osteo isn't powergaming. If you give a five unit injection for one broken bone, or two, etc, yeah, that isn't power gaming. **But nobody fuckin' does that**

I've already talked to Cadyn about bicaridine, I overlooked it at first. 

It is a playerbase issue, but half of the time, and I'm sure you understand this, there is no need to rush. There is a reason why I call it 'crutch chemming'. They get used to the quiet, relaxed parts of medical, give nothing but peri injections, etc. Then the **second** an actual emergency happens, 3+ declining patients, that sort of thing, their peri becomes useless, they have no idea what to do. Or if they don't have osteo, or myel, etc. I've run into medical doctor players here who don't know how to fix bones, after playing weeks as doctor. The point of this is to pull the carpet out from under them; they will either stop treating medical like some chore, or they won't and they'll find the role to not be for them. 

Convincing the playerbase is not that easy. I played doctor for a good while, did things like nothing but field surgery for a real week, etc, I built up a bit of a reputation before hopping into the CMO role. Even with that, getting people to apply what you tell or teach them isn't as simple as saying that surgery is easy. And the OD comment isn't valid because I should have explained what I meant by 40 units of peridaxon. 

They have no reason to rely on surgery, or organ chems that are more effective than peri. As said, they seem to see medical as a chore, and focus on the roleplay/vorny aspects of it. Thats all well and good, _until someone else's shift gets fucked because you don't know what you're doing._ Teaching them isn't that easy, they have to WANT the knowledge you're teaching them. That, and surgery is an experience thing. Sure, I can teach someone the steps, but, I know that I had a period in which I relied on the wiki for surgeries, it took a hot minute to remember the steps and what not. 

I'm just aware of who you are because a lot of folk talk about said guide, honestly. That and, its a playerbase problem is all, so medical knowledge isn't that useful in this case. I meant and continue to mean no offense.

**Comment by Wickedtemp** [comment]
The short version is, that isn't going to be fixed. This type of thing happens on Virgo, too. There have been medical players who'd been around for like a year, and they still can't handle a few surgeries without nearly killing patients.

If you want to create an environment where they have to improve and stop relying on crutches, you have to have more emergency situations.

**Comment by LunarFleet** [comment]
Not really. If the softer, more lazy solutions are removed, they're going to get used to the tools used in emergencies, outside of emergencies. If people cannot rely on osteo to fix bones normally, they're not going to rely on it in emergencies, and they're not going to not know what to do in emergencies, because it is the same as what they do normally.

**Comment by Docosahexaenoic-Acid** [comment]
> Myelamine no longer heals internal bleeding, instead temporarily slowing bleeding bleeding by a factor of 90% on average, 

Inaprovaline already does this. 

> Osteodaxon no longer mends fractures, instead it temporarily negates the effects of all fractures, allowing the user to use such organs as if they were not broken.

I would suggest to check splint n' organ's code to see if you're not just making splints in temporary liquid form on some limbs.

> Peridaxon no longer heals organs, instead it reverses any necrotic organs, both internal external

Normal spaceacillin is also said to be-able to do this, but check the code for it just incase.

**Comment by Darlantanis** [comment]
So, brief feedback. I'm not reading the mess of comments yet because I have limited time before I need to go to bed, so some of this has likely already been brought up.

Osteodaxon
This needs nerfed. Whether it needs nerfed to require a certain dose to mend bones, needs to not heal bones at all, or a rework of how bone breaking works is something I'm undecided on. The implementation in this PR however I'm not a fan of. This is chemical splints but without the slowdown of leg/feet splints and the ability to "splint" the torso/head. I don't have a suggestion yet though as a side note I could see this chem being useful as an addition rather than a replacement for osteo.

Myleamine
No. This does not currently need a nerf nor is it viable to make IB treatable by surgery only. IB is often the death of exploration despite the existence and availability of myleamine in its current state. This is also just turning it into inapravoline.

Peridaxon
This doesn't need a nerf either. It's an expensive chemical with low healing and annoying side effects. Frankly the only reason it's meta is because for the longest time it was the only chemical capable of healing internal organs (excluding brain and liver).

As for the PR itself, this seems less like an attempt at balancing chemicals and more of an attempt to force/incentivize surgery. It is perfectly to prefer surgery over chems just as it is perfectly fine to prefer chems over surgery. Trying to force one over the other is not, particularly given how pervasive these damages can be throughout the game. You can't assume a knowledgeable doctor is always going to be available when somebody gets hurt.

**Comment by LunarFleet** [comment]
First, to say that inaprovaline already does that is like saying that a pistol is the same as a cannon. They both shoot metal projectiles, yes, but one is far more powerful than the other. Looking at the code, inaprovaline halves bleeding, changed myel reduces bleeding **to** 10%

I don't know if it negates splint slowdown or not, that is simply what I suggested, up to Cadyn whether or not that was implemented correctly.

Going on to Nadyr...

The change to myelamine is not a nerf for all intents and purposes. As you said, and as I can agree with, IB is the death of many expeditions. If you can just spend some phoron to make a bottle or two of myelamine, allowing you to ignore IB and continue exploring until completed, I would not say that is a nerf. And even if that is still deemed as a nerf, then very well, perhaps combine the suggested and past mechanics of myel, making it slow bleeding and heal IB, at the same rate it usually does. Myel is not as bad as the other two chemicals in the changes. 

The problem with peri is that, for whatever reason, it is the magic cure all chem, despite there being multiple other chems which are specifically designed for healing organs. It invalidates multiple other chems to the point that 9/10 medical players I ask haven't even heard the word 'Gastirodaxon' or 'respirodaxon' before. This is not about forcing people to do surgery, it is about not invalidating 3+ other chems. **Also,** Peridaxon is **not** expensive, unless the recipe has changed and I have not noticed, which I doubt. Indeed, taking a glance at even the wiki for a moment will show that the only 'cost' of peri is that it takes four phoron for 60 units of peri. 60 units, thats enough to heal 60 damage to every organ. Now, lets put this into perspective of the OTHER organ healing chems, specifically the unheard of ones, which heal groups of organs related to their name. 

Respirodaxon takes 2 parts dexalin plus, two parts biomass, one part phoron. Alright, to make 60 units of Respirodaxon, you need 13 units of phoron. Remember, this only heals the larynx, lungs, and... whatever the hell the gbladder is, thanks code. 

Gastirodaxon heals... whatever the fuck this list is 'O_APPENDIX, O_STOMACH, O_INTESTINE, O_NUTRIENT, O_PLASMA, O_POLYP'
And once more, 60 units of Gastirodaxon should cost 13 phoron, once again. Versus Peridaxon's four, very cool.

There are more, and their costs are about the same; more expensive than peridaxon, by a lot. Which, because peridaxon is cheaper, that ultimately ends up making it more effective at healing organs too. It doesn't matter if it only repairs organs at a 1-1 ratio if you can make 3x the amount with the same amount of phoron. (Its 6 am so, if my math sounds wrong, please double check it if it sounds off, I know I will do so in the morning if someone else doesn't)

And lastly, there are still alternatives to IB and broken bones, such as Agent A/B/C from science. There 'not always being knowledgeable doctors' should not be the excuse for reliance on near powergamey chemicals, mostly referring to peri and osteo, less so myel. The bare minimum of medical knowledge should not be 'knowing how to inject magic heal all-chem' it should be knowing how to fix the problem yourself. Fancy chems are a luxury, not something to be relied upon, especially when there are alternatives like **specialized** organ healing chems.

**Comment by cadyn** [comment]
Just gonna put this out there: I've read most of these comments, and I like that there's a lot of open discussion about this. The fact that I made this PR does not necessarily mean that I agree with all the changes being made, in fact, I'm mostly neutral about them, however, I am willing to make changes as needed once this discussion comes to a consensus(unlikely), or I receive a decision from a headmin/a staff team consensus (more likely).

**Comment by LunarFleet** [comment]
Also, cause I just realized this, I wanna mention it, very shortly. 

There already are things in medical which are not double sided in how they can be cured. For example, surgery cannot fix oxygen or toxin damage. Having it so bones cannot be fixed with chemistry would be the same way.

**Comment by Darlantanis** [comment]
The point of myleamine is it's a simple fix to IB obtainable without medical/given by medical so you don't need to stop and do surgery (plus by antags). You can get it from cargo or from dragging a spider corpse to somewhere with chemicals (you're also supposed to be able to buy clotting injectors from exploration vendors but I've just discovered that tweak was mistakenly overwritten with an upstream PR). 

If your issue with peri is it outclassing specialized organ chems, buff the specialized chems. 

The time and RNG required to obtain slime agents make them poor alternatives to current chems. They're most useful for a xenobiologist fixing injuries they've incurred raising slimes.

The lack of medical alternatives to treating oxy/tox is irrelevant. If you want alternatives, code them.

>The bare minimum of medical knowledge should not be 'knowing how to inject magic heal all-chem' it should be knowing how to fix the problem yourself. Fancy chems are a luxury, not something to be relied upon, especially when there are alternatives like **specialized** organ healing chems.
A player pushing for code changes to enforce a way they believe other players **should** play a department is a massive red flag when I'm reviewing pull requests. At best this is well-intended but likely to alienate many medical players; at worst, it's elitism.

No support.

**Comment by LunarFleet** [comment]
I figured my suggestion for myel to be more of a buff than a nerf, especially given the fact that I've had times where multiples injections of myel did not fix the problem. But if you insist, alright then.

Xenobiology was just an example, as were treating oxy/tox, the last two being a point that there are already mechanics without alternatives, forcing you into a play style. 

Regardless however, as said before, the idea arose from a playerbase issue; plain and simple, it pains me, and seems to annoy patients, when doctors don't really seem to know what they are doing. Being stuck in medical is not fun for anyone, it interrupts what you were already doing, often with something as annoying and non-inclusive as being stuck unconscious. I don't care if others use my style of medical, and honestly, I don't really think they should; I refuse science tools, chemicals beyond the most bare basics, etc, because I enjoy the challenge. I don't want people to do that, I just want people to treat medical as something more than a mere chore, figured working through the chemicals which result in the most carelessness would be a good start.

**Comment by Darlantanis** [comment]
As per staff discussion this is not supported in its current state. That combined with my talk with LunarFleet tells me any attempted balancing will be tackled in a different PR.
